### ISSUE TITLE
tanh

### ISSUE THREAD

**Issue by xunguangwang** [jiangqy/DPSH-pytorch#10]
@jiangqy hi, doctor jiang! 
On the basis of DPSH, I added tanh as the last layer function of the network, and the loss suddenly increased during training, as shown below; when I removed tanh, everything was normal. Do you know what happened?
Epoch:  34/150  Train_loss: 0.23092
Epoch:  35/150  Train_loss: 0.22569
Epoch:  36/150  Train_loss: 0.22112
Epoch:  37/150  Train_loss: 0.21693
Epoch:  38/150  Train_loss: 3.01185
Epoch:  39/150  Train_loss: 4.22574
Epoch:  40/150  Train_loss: 5.80905
Epoch:  41/150  Train_loss: 9.34127
Epoch:  42/150  Train_loss: 4.26501
Epoch:  43/150  Train_loss: 5.96325
Epoch:  44/150  Train_loss: 5.72492
Epoch:  45/150  Train_loss: 6.11856

**Comment by swuxyj** [comment]
Maybe it's just an accident during the optimization process. I didn't encounter this situation after adding tanh 🥇

**Comment by jiangqy** [comment]
I think you can tune the learning rate to fix this problem.
